qsym/executor.py

```diff
@@ -65,6 +65,14 @@ def status_file(self):
     def log_file(self):
         return os.path.join(self.testcase_dir, "pin.log")
 
+    def check_elf32(self):
```

Comment:
Do you think it would be fine?  or should we use other elf parsing library?

Reply:
I looked at the elf parsing libraries, and it seemed like overkill to add another dependency just to support this simple check.  I also thought about using the command line utility `file` and parsing the output, which would also work and is almost always available.  In the end I preferred the simplicity of this approach.  All _properly_ compiled ELF binaries will be compliant.

Reply:
Agree. I think it's better to use this approach. Then if we need more functionality about ELF parsing later, we can use others.